embassy-stm32f4/src/serial.rs

```diff
@@ -199,7 +196,7 @@ impl Serial<USART1, Stream7<DMA2>, Stream2<DMA2>> {
             let mut rx_transfer = Transfer::init(
                 rx_stream,
                 usart,
-                rx_buffer,
+                buf.static_write_buffer(),
```

Comment:
Error:

```
the trait bound `(*mut u8, usize): core::ops::DerefMut` is not satisfied

the trait `core::ops::DerefMut` is not implemented for `(*mut u8, usize)`

note: required because of the requirements on the impl of `embedded_dma::WriteBuffer` for `(*mut u8, usize)`
note: required because of the requirements on the impl of `embedded_dma::StaticWriteBuffer` for `(*mut u8, usize)`rustc(E0277
```